.clang-format

```diff
@@ -0,0 +1,17 @@
+# A clang-format style that approximates Python's PEP 7
+# Useful for IDE integration
+BasedOnStyle: Google
+AlwaysBreakAfterReturnType: All
+AllowShortIfStatementsOnASingleLine: false
+AlignAfterOpenBracket: Align
+BreakBeforeBraces: Stroustrup
+ColumnLimit: 79
+DerivePointerAlignment: false
+IndentWidth: 4
+Language: Cpp
```

Comment:
Why Cpp instead of C?

Reply:
C is not a valid option, Ctrl+F for "LanguageKind" on this page, where it recommends Cpp for formatting C code: https://clang.llvm.org/docs/ClangFormatStyleOptions.html